examples/mobile/src/lib.rs

```diff
@@ -11,17 +12,25 @@ use bevy::{
 #[bevy_main]
 fn main() {
     let mut app = App::new();
-    app.add_plugins(DefaultPlugins.set(WindowPlugin {
-        primary_window: Some(Window {
-            resizable: false,
-            mode: WindowMode::BorderlessFullscreen(MonitorSelection::Primary),
-            // on iOS, gestures must be enabled.
-            // This doesn't work on Android
-            recognize_rotation_gesture: true,
-            ..default()
-        }),
-        ..default()
-    }))
+    app.add_plugins(
+        DefaultPlugins
+            .set(LogPlugin {
+                level: Level::DEBUG,
+                filter: "wgpu=error,bevy_render=info,bevy_ecs=trace".to_string(),
```

Comment:
```suggestion
                // This will show some log events from Bevy to the native logger.
                level: Level::DEBUG,
                filter: "wgpu=error,bevy_render=info,bevy_ecs=trace".to_string(),
```

Reply:
Thanks